TTS/api.py

```diff
@@ -0,0 +1,177 @@
+from pathlib import Path
+from TTS.utils.synthesizer import Synthesizer
+from TTS.utils.manage import ModelManager
+
+
+class TTS():
```

Comment:
I don't know If calling it TTS is the best thing because it conflicts with the name of the package so it might prevent this kind of import: `from TTS.api import TTS`

Reply:
I don't get why you can't import like that.

I'm also open for suggestions 👍

Reply:
Then you override TTS, might be annoying if you plan on importing other stuff from TTS. But yeah not super important.